src/NzbDrone.Core/Indexers/HttpIndexerBase.cs

```diff
@@ -66,6 +66,18 @@ public override Task<IndexerPageableQueryResult> Fetch(MovieSearchCriteria searc
                 return Task.FromResult(new IndexerPageableQueryResult());
             }
 
+            var caps = GetCapabilities();
+
+            if ((!caps.MovieSearchImdbAvailable && searchCriteria.ImdbId.IsNotNullOrWhiteSpace()) ||
```

Comment:
If not movie imdb search is true AND if imdbid is populated

IMDb is supported true => false & id is populated; outcome is false, skip search

the negate seems to be wrong?

imdb is not supported inverts to true & IMDb has value; outcome is true and search?

(pre-coffee)